README.md

```diff
@@ -4,3 +4,20 @@ A version of [font-roboto](https://github.com/PolymerElements/font-roboto) using
 locally available font files
 
 The font files are subject to the Apache License found here: https://github.com/google/fonts/blob/master/apache/roboto/LICENSE.txt
+
+## Usage in a Polymer 2.x Application
+
+Add the following line to the end of your `bower.json`
```

Comment:
Is this accurate? IINM, the line needs to go into the `dependencies` section specifically, and this would normally be done using a `bower install --save` command...yes, this seems to work for me:

```
$ bower install --save 'font-roboto=PolymerElements/font-roboto-local#^1.1.0'
```

Reply:
You are right. I missed to mention that the line goes into the `dependencies` section. Should I create another pull request?

Reply:
I have no authority here, so it's up to you. I'd probably just force push to your existing branch, but anyway, I hope you don't think I have permissions to merge this since I don't... :/

Reply:
Alright. Thanks. Let's wait for someone with authority to merge it.

At least someone wanting to using roboto locally will eventually find this pull request and this discussion. :D

Reply:
You don't need to create another PR, you can just add another commit to this one (locally just do a `git commit -am "your new message"; git push`. You don't have to force push either, i can squash the commits if we merge this.

I agree with @davidmaxwaterman that your line is incorrect. I would prefer something like:

To add this to your application, run this command from your project root directory:
```
bower install --save 'font-roboto=PolymerElements/font-roboto-local#^1.1.0'
```
This will add and install the dependency to your project. Then, you can import the file and use the font in your page:

```html
<head>
  ...
  <link rel="import" href="roboto.html">
  <style>
    .roboto {
      font-family: Roboto
    }
  </style>
</head>
<body>
  ...
  <div class="roboto">This is using Roboto!</div>
</body>
...
```

Reply:
@notwaldorf Thank you. I've made the changes required.